tasks/task02/triangle_area.py

```diff
@@ -0,0 +1,56 @@
+import math
+
+
+def get_values(data: str):
+    values = data.split(' ')
+    first_v = int(values[0])

```

Comment:
It seems this part could be rewritten as 
```
def get_values(data:str):
   return map(int, data.split())
```

Reply:
Rewrited